### PR TITLE
bug/python-version-compibility-issues

### PR DESCRIPTION
**Pull request by WinnieXU1**
if python version is not 2.7, warn and quit

### CODE REVIEW

File: infrasim/cli.py [hunk 1]
```diff
@@ -299,6 +299,11 @@ def add_command_parsers(subparser):
 
 
 def command_handler():
+    py_version = sys.version_info
+    if not (py_version[0] is 2 and py_version[1] is 7):
+        print "Warning: please install Python 2.7 first."
+        return
+
```
Comment (turtle-fly) [inline infrasim/cli.py]: Can we solve it from pip's perspective? @XiaowenJiang

Comment (XiaowenJiang) [inline infrasim/cli.py]: @turtle-fly How to solve it from pip = =? You mean there would be a library to adjust grammar among different python versions?

Comment (turtle-fly) [inline infrasim/cli.py]: Can pip require infrasim-compute to be used >= python 2.7

**Comment by fub2** [comment]
2 points here:
1. is it supposed to fix issue in fru-gen and bmc data capturing and generating tool? But now it is updating code in infrasim-compute?

2. In setup.cfg
```
classifier =
    Development Status :: 5 - Production/Stable
    Environment :: Console
    Intended Audience :: Information Technology
    License :: OSI Approved :: Apache Software License
    Operating System :: POSIX :: Linux
    Programming Language :: Python
    Programming Language :: Python :: 2
    Programming Language :: Python :: 2.7
```
So seems we don't need to worry about that somebody is running infrasim-compute in a none python-2.7 environment.

**Pull request closed without merging**